internal/config/profile.go

```diff
@@ -225,18 +225,41 @@ func (p *profile) SetOrgID(v string) {
 	p.Set(orgID, v)
 }
 
+// GetConfigDescription returns a map describing the configuration
+func GetConfigDescription(name string) map[string]string {
+	redactedValue := "redacted"
+
+	m := viper.GetStringMapString(name)
+
+	if _, ok := m[privateAPIKey]; ok {
+		m[privateAPIKey] = redactedValue
+	}
+
+	if _, ok := m[publicAPIKey]; ok {
+		m[publicAPIKey] = redactedValue
+	}
+
+	return m
+}
+
 // Load loads the configuration from disk
-func Load() error { return p.Load() }
-func (p *profile) Load() error {
+func Load() error { return p.Load(false) }
```

Comment:
Found why, this should be true
```suggestion
func Load() error { return p.Load(true) }
```
let's fix this or revert all this

Reply:
ha, good spot. Of course it was just something simple